package.json

```diff
@@ -72,8 +77,22 @@
     "react-app-rewired": "^1.4.0",
     "react-dom": "^16.2.0",
     "react-hot-loader": "^3.1.3",
-    "react-markdown-loader": "git+https://github.com/jamesmfriedman/react-markdown-loader.git#master",
+    "react-markdown-loader":
+      "git+https://github.com/jamesmfriedman/react-markdown-loader.git#master",
```

Comment:
A particular reason for the new lines for some key values? Linting? Auto format? To me it looks nicer if everything is a one liner, especially since it doesn't really save much horizontal space anyway in most cases.